app/jobs/persist_vacancy_get_more_info_click_job.rb

```diff
@@ -0,0 +1,8 @@
+class PersistVacancyGetMoreInfoClickJob < ApplicationJob
+  queue_as :page_view_collector
```

Comment:
Should we change this queue name to something like `persist_counters` or something similar/generic?

Reply:
Yep, I thought that at the time, but forgot to do it!